### ISSUE TITLE
It just crashes

### ISSUE THREAD

**Issue by lostdoge08** [hinyb/hinyb-Dropability#1]
When I try to launch the game with this mod, the game crashes. I've even tried running the game with JUST this mod and it's dependencies and the game still crashes, even tried going to a few older versions and had the same issue.

Here's what I believe is the important part of the log (github character limit):

```
[15:39:46.6921432][ERROR/Code_Function_GET_the_function.hpp:151] last code_execute: gml_Object_oInit_Alarm_9
[15:39:46.6921504][WARNING/log.cpp:28] MGReturns-ENVY: stack traceback:
	[string "log.warning(debug.traceback())"]:1: in main chunk
	[C]: in function 'base.require'
	...es\eh\ReturnOfModding\plugins\hinyb-Dropability\main.lua:18: in main chunk
```

And the full log is listed below:

[LogOutput.log](https://github.com/user-attachments/files/18215616/LogOutput.log)

**Comment by hinyb** [comment]
Oops, I forgot to update this mod.
This issue is caused by an update to ReturnOfModding; the dynamic_hook_mid function no longer requires some parameters.
I will fix this in the next update, possibly tomorrow. If you want to use the mod before I update it, please downgrade the version of ReturnOfModding.

**Comment by hinyb** [comment]
If you don't have any issues, I will close this.

**Comment by hinyb** [comment]
It should be fixed in v3.1.5 now. If you still encounter any issues, please reopen it.
